ecllibrary/teststd/uni/TestEndsWith.ecl

```diff
@@ -36,7 +36,10 @@ EXPORT TestEndsWith := MODULE
     //Check action on a string containing different variations/combinations of numbers and other characters.
     EXPORT Test24 := ASSERT(Uni.EndsWith('123abc 23.6 abc123','23.6 abc123','') = TRUE);
     //Test other space characters (< 0x20).
-    EXPORT Test26 := ASSERT(Uni.EndsWith('\nt\t','t','') = FALSE);
+    //It appears the unicode trim semantics are to remove whitespace characters 0 that was not the intention, it
```

Comment:
Is the '0' in this comment a typo for - ?

Not sure that the fact that TRIM(unicode) is wrong necessarily means that Uni.EndsWith() should be wrong?

Reply:
yes 0 must have replaced the text.
The trim function is also used by the edit distance - at least it is consistent.  @richardkchapman Should I change it, or leave it as it is?

Reply:
conclusion is that trim should not remove arbitrary whitespace.  That should be fixed, and another function added which does trim whitespace (and is also available for strings).